### DISCUSSION TITLE
list of teminals that are compatibe

### DISCUSSION THREAD

**Discussion by cruzecontrol121** [GitSquared/edex-ui#1002] [Questions] [answered]
i was just wondering out of curiosity if there was a list of terminals that it was compatible with.   i do a lot of terminal based things and i was hoping i could use something different on my windows pc.    any and all help is greatly appreciated

**Comment by Animus-Surge** [comment]
I believe most terminals work with eDEX. I've been able to solely run NodeJS in it before, so it shouldn't be any different to the git cli or something like that. I know of bash, cmd, powershell, and a few others being usable with it. Worth checking out other options, though, and seeing if they work.

**Comment by GitSquared** [comment, marked as answer]
Anything that works with standard stdin/out/err pipes exchanging UTF-8 text should be capable of running in eDEX.
